recipes/filterpy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "filterpy" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "42d42d7aeef752e297839776ae01721e79bf446e3c17e3d48914c03b5fbdd816" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    # the archive uploaded to pypi is missing the LICENSE file, this patch adds it back
+    - license.patch
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - matplotlib
+    - scipy
```

Comment:
No it's just a source install. I will remove these from the build requirements. Thanks for the  suggestion.